src/Conf.ml

```diff
@@ -269,9 +282,22 @@ let info =
     ; `P
         "Unless otherwise noted, any option \
          $(b,--)$(i,option)$(b,=)$(i,VAL) detailed in this section can be \
-         set in an $(b,.ocamlformat) configuration file with an \
-         '$(b,option = )$(i,VAL)' line, or with an $(b,[@@@ocamlformat \
-         \")$(i,option)$(b,=)$(i,VAL)$(b,\"]) attribute." ]
+         set in many ways, its value is determined in the following order \
+         (of increasing priority): the default value is used if no other \
+         value is specified, the value can be modified in an \
+         $(b,.ocamlformat) configuration file with an '$(b,option = \
+         )$(i,VAL)' line (*), or using the OCAMLFORMAT environment \
+         variable: \
+         $(b,OCAMLFORMAT=)$(i,option)$(b,=)$(i,VAL)$(b,,)...$(b,,)$(i,option)$(b,=)$(i,VAL), \
+         or as an optional parameter on the command line, or with a global \
+         $(b,[@@@ocamlformat \")$(i,option)$(b,=)$(i,VAL)$(b,\"]) \
+         attribute in the processed file, or with an $(b,[@@ocamlformat \
+         \")$(i,option)$(b,=)$(i,VAL)$(b,\"]) attribute on expression in \
+         the processed file."
+    ; `P
+        "(*) .ocamlformat files in current and all ancestor directories of \
+         the input file are used, with more distant ancestors having lower \
```

Comment:
Actually, maybe s/of the/for each/ would be clearer in the case that there are multiple input files.